_component_design/popovers.md

```diff
@@ -14,81 +14,171 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Popovers are triggered as a result of a user action and provide a user with
-additional context through language, actions through buttons, or both. Popovers
-come in a the following types Hover Popover, Triggered Popover, and Closeable
-Popover
+## Introduction
+{: .hxSectionTitle}
```

Comment:
Remove classes per discussion in https://github.com/rackerlabs/design-system/pull/292#issuecomment-338716258

Reply:
removed all instances across the doc.